libs/ledger-live-common/src/featureFlags/helper.tsx

```diff
@@ -0,0 +1,10 @@
+// Recover compatibility check function, based on feature and device
+export function checkRecoverCompatibility(feature, deviceModelId) {
```

Comment:
I think that you can rename the function to remove the comment above with something like:
`checkRecoverAvailabality()` or `isRecoverDisplayed()`

`Compatibility` makes us think that it's only related to the device but it's also related to the feature flag.